modules/data/src/main/scala/scaladex/data/central/CentralMissing.scala

```diff
@@ -162,39 +156,14 @@ class CentralMissing(paths: DataPaths)(implicit val system: ActorSystem) {
     DownloadRequest(groupId, artifactId, version, created)
   }
 
-  def savePomsAndMeta(pom: PomContent): Unit = {
-    val sha1 = Sha1(pom.content)
-    val repository = LocalPomRepository.MavenCentral
-
-    val meta = Meta(sha1, pom.artifact.path, pom.artifact.created)
-
-    // write meta
-    Meta.append(
-      paths,
-      meta,
-      repository
-    )
-
-    // write pom
-    val pomPath = paths.poms(repository).resolve(s"$sha1.pom")
-    Files.write(pomPath, pom.content.getBytes(StandardCharsets.UTF_8))
-  }
+  def savePomsAndMeta(pom: PomContent): Unit =
+    ???
 
   // data/run central /home/gui/scaladex/scaladex-contrib /home/gui/scaladex/scaladex-index /home/gui/scaladex/scaladex-credentials
   def run(): Unit = {
     val metaExtractor = new ArtifactMetaExtractor(paths)
     val pomsReader = new PomsReader(new CoursierResolver)
-    val allGroups: Set[String] =
-      pomsReader
-        .loadAll(paths.poms(LocalPomRepository.MavenCentral))
-        .flatMap {
-          case (pom, _) =>
-            metaExtractor
-              .extract(pom)
-              .filter(meta => meta.platform != Platform.Java)
-              .map(_ => pom.groupId)
-        }
-        .toSet
+    val allGroups: Set[String] = ???
```

Comment:
is that ok ?

Reply:
It's done on purpose.

This code won't be useful anymore but I would like to keep for a while to translate it into a scheduled job. That would be the scheduled job that find the missing artifacts from Maven Central.

Reply:
ok!